assembly-combined-package/assembly-combined/sbin/linkis-stop-all.sh

```diff
@@ -86,6 +86,37 @@ SERVER_NAME="ps-publicservice"
 SERVER_IP=$PUBLICSERVICE_INSTALL_IP
 stopApp
 
+#datasourcemanager
+SERVER_NAME="ps-datasourcemanager"
+SERVER_IP=$DATASOURCEMANAGER_INSTALL_IP
+stopApp
+
+#metadatamanager
+SERVER_NAME="ps-metadatamanager"
+SERVER_IP=$METADATAMANAGER_INSTALL_IP
+stopApp
+
+#metadatamanager-es
+SERVER_NAME="ps-metadatamanager-es"
+SERVER_IP=$METADATAMANAGER_ES_INSTALL_IP
+stopApp
+
+#metadatamanager-hive
+SERVER_NAME="ps-metadatamanager-hive"
+SERVER_IP=$METADATAMANAGER_HIVE_INSTALL_IP
+stopApp
+
+#metadatamanager-kafka
+SERVER_NAME="ps-metadatamanager-kafka"
+SERVER_IP=$METADATAMANAGER_KAFKA_INSTALL_IP
+stopApp
+
+#metadatamanager-mysql
+SERVER_NAME="ps-metadatamanager-mysql"
+SERVER_IP=$METADATAMANAGER_MYSQL_INSTALL_IP
+stopApp
```

Comment:
So many DataSource process, I suggest create a new stop DataSources script to stop it.

Reply:
thanks，I will fix it later